### DISCUSSION TITLE
post/azure-cli-service-principal-roles-groups

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#78] [Announcements]
# post/azure-cli-service-principal-roles-groups

Mark Heath's Development Blog

https://markheath.net/post/azure-cli-service-principal-roles-groups

**Comment by maksimsKravcevs** [comment]
What if I assign specific  app role to group and then find groups assigned to specific sp app roles

**Reply by maksimsKravcevs** [reply]
Is there some az ad  cli commands to do it
